### PR TITLE
add original numeric id field to FAQArticle and FAQCategory

### PR DESCRIPTION
**Pull request by vborszeki**

### CODE REVIEW

File: src/FAQ/dataloaders/getFAQCategory.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { get } from '../../common/services/HttpRequest';
 import type { FAQCategoryType } from '../types/outputs/FAQCategory';
 
 export type FAQArticleItem = $ReadOnly<{|
-  id: string,
+  id: number,
```
Comment (git-toni) [inline src/FAQ/dataloaders/getFAQCategory.js]: You should rather keep the `id` as a string and add a new field to this type called `databaseId` with number type.

File: src/FAQ/queries/__test__/FAQCategory.test.js [hunk 2]
```diff
@@ -15,15 +15,16 @@ describe('allFAQCategories', () => {
     const resultsQuery = `query FAQSubcategories($id: ID!) { 
       FAQCategory(language: en, id: $id) {
         id
+        originalId
```
Comment (git-toni) [inline src/FAQ/queries/__test__/FAQCategory.test.js]: The "consensus" in the repo is to call this field `databaseId` if I'm not mistaken. Pls rename this occurrence and all the others in the diff :slightly_smiling_face:

Comment (vborszeki) [reply src/FAQ/queries/__test__/FAQCategory.test.js]: So which version should we stick with then?

Comment (jaroslav-kubicek) [reply src/FAQ/queries/__test__/FAQCategory.test.js]: `originalId` is fine IMHO :)

**Comment by jaroslav-kubicek** [comment]
Looks great, thanks. Pls just rebase, thx @vborszeki

File: src/FAQ/queries/__test__/FAQCategory.test.js (hunk 2 above)
Comment (mrtnzlml) [reply src/FAQ/queries/__test__/FAQCategory.test.js]: `id(unmasked: true)` 😊

Comment (git-toni) [reply src/FAQ/queries/__test__/FAQCategory.test.js]: > Hotels have originalId, Booking has databaseId so in fact there is no consensus
Ouch. :smile:

File: src/FAQ/dataloaders/getFAQCategory.js (hunk 1 above)
Comment (git-toni) [reply src/FAQ/dataloaders/getFAQCategory.js]: :man_facepalming:

**Pull request merged**